docs/admin/access.rst

```diff
@@ -139,8 +130,7 @@ The authentication models consist of several objects:
     Individual permissions defined by Weblate. Permissions can not be
     assigned to users. This can only be done through assignment of roles.
 `Role`
-    A Role defines a set of permissions. This allows reuse of these sets in
-    several places, making the administration easier.
+    A role defines a set of permissions, and can be reused in several places.
```

Comment:
```suggestion
    A role defines a set of permissions, for ease of use in several places.
```

Reply:
I think both are valid clauses here.
If I want to remember what an old group did, I can keep the set of privileges it had around.

Reply:
May be a more straightforward variant:
```suggestion
    A role defines a set of permissions, so it can be reused in different groups.
```